casdoorsdk/user.go

```diff
@@ -233,6 +238,57 @@ func GetUserByUserId(userId string) (*User, error) {
 	return user, nil
 }
 
+func UpdateUserPassword(owner, name, oldPassword, newPassword string) (bool, error) {
+	param := map[string]string{
+		"userOwner":   owner,
+		"userName":    name,
+		"oldPassword": oldPassword,
```

Comment:
The user said:

`
side note: is there a chance for you to add a flexible function to send http POST requests, similar to how the SDK has DoGetBytesRaw for http GET requests to the casdoor server.
`

Can we support this?